### PR TITLE
[Cli] add db restore to aptos cli

### PR DESCRIPTION
**Pull request by areshand**

### CODE REVIEW

File: crates/aptos/src/node/mod.rs [hunk 1]
```diff
@@ -1486,65 +1488,27 @@ impl CliCommand<()> for AnalyzeValidatorPerformance {
...
-        // hack: get around this error, related to use of `async_trait`:
-        //   error: higher-ranked lifetime error
-        //   ...
-        //   = note: could not prove for<'r, 's> Pin<Box<impl futures::Future<Output = std::result::Result<(), CliError>>>>: CoerceUnsized<Pin<Box<(dyn futures::Future<Output = std::result::Result<(), CliError>> + std::marker::Send + 's)>>>
+        let storage = self.storage.init_storage().await?;
         tokio::task::spawn_blocking(|| {
             let runtime = tokio::runtime::Runtime::new().unwrap();
-            runtime.block_on(RestoreCoordinator::new(opt, global_opt, storage).run())
+            runtime
+                .block_on(RestoreCoordinator::new(self.opt, self.global.try_into()?, storage).run())
```
Comment (banool) [inline crates/aptos/src/node/mod.rs:1511]: I wonder if this is still necessary, vs just awaiting the inner function directly? If it _is_ still necessary, can you please leave that comment from before explaining the hack.

**Comment by 0xjinn** [comment]
Though this seems more internal to me, I wonder if we should update the changelog to reflect this? cc: @banool Thoughts?

**Review by msmouse — APPROVED** [review]
❤️  LGTM. Address others' concerns please.

File: crates/aptos/src/node/mod.rs (hunk 1 above)
Comment (areshand) [reply crates/aptos/src/node/mod.rs:1511]: still necessary. Otherwise same compiler error with just inner function. added back the comment

**Comment by areshand** [comment]
> changelog

updated the changelog.

**Review by 0xjinn — APPROVED** [review]
LGTM.

Please ping me and let me know if this is urgent, I can start a release today if needed.

**Comment by areshand** [comment]
> LGTM.
> 
> Please ping me and let me know if this is urgent, I can start a release today if needed.

will let you know once it lands, a release today would be timely needed. Thank you.

**Pull request merged**